client/src/components/body/EducatorResources/EducatorResources.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import WhatsAvailable from "./WhatsAvailable";
+import ResourcesHeader from "./ResourcesHeader";
+import ResourcesSection from "./ResourcesSection";
+
+const EducatorResources = () => {
```

Comment:
This component seems small. Can/should we add the other components into this file?

Reply:
Do you mean the 'What's Available' and the header?